### ISSUE TITLE
False positive in Rails/FilePath cop

### ISSUE THREAD

**Issue by lucascaton** [bbatsov/rubocop#3947]
## Expected behavior

`Rails/FilePath` cop ([PR](https://github.com/bbatsov/rubocop/pull/3822)) shouldn't complain when I have the following code:

```ruby
system "rm -rf #{Rails.root.join('public', 'system')}"
```

## Actual behavior

`$ rubocop --only Rails/FilePath`

```ruby
Offenses:
lib/tasks/db.rake:52:12: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
    system "rm -rf #{Rails.root.join('public', 'system')}"
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## RuboCop version

```
0.47.1 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin16)
```

Thank you.
